src/Viewer/Viewer.js

```diff
@@ -232,7 +230,9 @@ export function Viewer ({fileInfo, prettifyLog, logEventNumber}) {
     window.onhashchange = () => {
         const urlHashParams = new VerbatimURLParams(window.location.hash, "#");
         const logEventIdx = urlHashParams.get("logEventIdx");
-        changeState(STATE_CHANGE_TYPE.logEventIdx, {logEventIdx: logEventIdx});
+        if (isNumeric(logEventIdx)) {
```

Comment:
We should have an else to reset the logEventIdx like in the other places, right?

Reply:
Done.